packages/patch/project.bri

```diff
@@ -0,0 +1,66 @@
+import nushell from "nushell";
+import * as std from "std";
+
+export const project = {
+  name: "patch",
+  version: "2.8",
+};
+
+const source = Brioche.download(
+  `https://ftp.gnu.org/gnu/patch/patch-${project.version}.tar.xz`,
+)
+  .unarchive("tar", "xz")
+  .peel();
+
+export default function patch(): std.Recipe<std.Directory> {
+  return std.runBash`
+    ./configure --prefix=/
+    make install DESTDIR="$BRIOCHE_OUTPUT"
+  `
```

Comment:
Interesting it doesn't need `make` to do the build first (the `README-hacking` file says to use it, but it seems to work fine and takes about the same amount of time to build either way)